src

```diff
@@ -0,0 +1 @@
+ecodan-ha-local
```

Comment:
Such a symlink will not be usable on Windows, I think instead the platformio.ini file should contain a:
```
[platformio]
src_dir=${PROJECT_DIR}/ecodan-ha-local
```
section, which would work on any platform.